index.html

```diff
@@ -104,9 +108,216 @@ <h2 class="color-heading" id="heading-2">
                 </a>
               </li>
             </ul>
+            <div class="downward arrow">
+              <img src="assets/images/Action.png" alt="" />
+            </div>
           </div>
         </div>
       </div>
     </section>
+
+    <section class="work-section">
+      <div class="opening-heading">
+        <h2 class="work-heading">My Recent Work</h2>
+      </div>
```

Comment:
- [ ] Change to `My Recent works` not My Recent work

Reply:
Have to add 's' at the end and should I make w a lowercase or just keep it in the upper case?

Reply:
Just add `s` at the end no need for `w` to be in the upper case